tests/WFC3_ecfs/S3_WFC3.ecf

```diff
@@ -37,7 +37,7 @@ bg_deg          0           # Polynomial order for column-by-column background s
 p3thresh        5           # X-sigma threshold for outlier rejection during background subtraction
 
 # Spectral extraction parameters
-spec_hw         5           # Half-width of aperture region for spectral extraction (relative to source position)
+spec_hw         8           # Half-width of aperture region for spectral extraction (relative to source position)
```

Comment:
My guess is that you changed this as you wanted this to be different than the background size, but now the background hw is smaller than the spec_hw which is dangerous

Reply:
Ugh, that was supposed to be bg_hw.